### PR TITLE
RCPS-141-stud-info-mgt-app:Automatically-exits-when-adding-student

### PR DESCRIPTION
**Pull request by Joven2002**

### CODE REVIEW

File: src/main/java/com/student/user/app/controllers/item/AddStudentController.java [hunk 1]
```diff
@@ -61,11 +63,13 @@ public class AddStudentController implements Initializable {
     @FXML
     private Button studentAddButton;
 
-    private StudentFacade studentFacade = new StudentFacadeImpl();
+
```
Comment (RoCS2024) [inline src/main/java/com/student/user/app/controllers/item/AddStudentController.java:69]: Use the StudentInfoMgtApplication class instead. DAOs should not be used in UI codes

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comment

File: src/main/java/com/student/user/app/controllers/item/AddStudentController.java (hunk 1 above)
Comment (Joven2002) [reply src/main/java/com/student/user/app/controllers/item/AddStudentController.java:69]: copy po thank you po

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
same comment

File: src/main/java/com/student/user/app/controllers/item/AddStudentController.java [hunk 2]
```diff
@@ -86,8 +91,17 @@ protected void onAddStudClicked(ActionEvent event) {
         } catch(Exception ex) {
             ex.printStackTrace();;
         }
+        Stage dashboardStage = new Stage();
```
Comment (zhairanicolepacheco) [inline src/main/java/com/student/user/app/controllers/item/AddStudentController.java]: implement the try/catch/finally statement

Comment (Joven2002) [reply src/main/java/com/student/user/app/controllers/item/AddStudentController.java]: okay copy

**Review by zhairanicolepacheco — APPROVED** [review]

**Review by RoCS2024 — APPROVED** [review]

**Comment by Joven2002** [comment]
https://meet.google.com/qdj-wrnq-jep

**Pull request merged**